app/Models/Review.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\Store;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class review extends Model
+{
+    use HasFactory;
+
+    public function review_id()
```

Comment:
@aaldemeery Do not add methods before attributes...take this down below the `$fillable` array